src/Compilers/Core/Portable/Operations/OperationInterfaces.xml

```diff
@@ -2944,6 +2949,13 @@
         <summary>The method that will be invoked to dispose the declared instances when pattern based disposal is used.</summary>
       </Comments>
     </Property>
+    <Property Name="DisposeArguments" Type="ImmutableArray&lt;IArgumentOperation&gt;" Internal="true">
```

Comment:
>DisposeArguments [](start = 20, length = 16)

I would expect the two test IOperation visitors that we have to be adjusted appropriately to verify the value in this property. Same for the property above. It probably makes sense to ignore an empty array in the visitor that dumps the tree in order to avoid changes in baselines for tests that do not have arguments.